src/main/java/uk/gov/ons/census/fwmt/csvservice/implementation/postcodeloader/LookupFileLoaderEventsConfig.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.ons.census.fwmt.csvservice.implementation.postcodeloader;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+
+@Configuration
+public class LookupFileLoaderEventsConfig {
+
+  public static final String POSTCODE_LOOKUP_LOADED = "POSTCODE_LOOKUP_LOADED";
+
+  @Bean
+  public GatewayEventManager addAddressCheckEvents(GatewayEventManager gatewayEventManager) {
```

Comment:
there is already an addAddressCheckEvents() in  the LookupFileLoaderEventsConfig class